request_a_govuk_domain/request/admin/email.py

```diff
@@ -67,7 +67,7 @@ def token(reference, domain_name: str) -> str:
     """
     # token_id is application reference except the prefix "GOVUK"
     token_id = reference[5:]
-
+    domain_lower = domain_name.lower()
```

Comment:
We do this already : https://github.com/co-cddo/domains-register-a-govuk-domain/blob/main/request_a_govuk_domain/request/forms.py#L156